scripts/ampHtmlValidator/index.js

```diff
@@ -58,7 +63,8 @@ const runValidator = async verbose => {
 
   const urls = pageTypes
     .map(pageType => getPageUrls({ pageType, environment, isSmoke }).flat())
-    .flat();
+    .flat()
+    .filter(url => !excludedUrls.includes(url));
```

Comment:
Each method: `map`, `flat`, `filter` will iterate the `pageTypes` array. It's not important here, but worth noting we would want to combine these operations to the first iteration if this was running in a user's browser. E.G.
```js
// flatMap combines mapping and flattening.
// Returning an empty array on L4 means this result is also flattened/removed/filtered.
const urls = pageTypes.flatMap((pageType) => {
  const url = getPageUrls({ pageType, environment, isSmoke }).flat();
  return excludedUrls.includes(url) ? [] : [url];
});
```